packages/util/src/index.ts

```
@@ -121,6 +164,26 @@ export function getInitialElementResults(
     }
+  } else if (
+    element.type === PrismaElementType.SELECTION &&
+    'answerCollection' in element &&
+    element.answerCollection &&
+    'entries' in element.answerCollection
+  ) {
+    const selections = element.answerCollection.entries.reduce<
+      Record<number, number>
+    >(
+      (acc, entry) => ({
+        ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)